draft-sheffer-wimse-s2s-protocol.md

```diff
@@ -136,6 +136,65 @@ This document uses "service" and "workload" interchangeably. Otherwise, all term
 
 ## Option 2: Authentication Based on HTTP Message Signatures
 
+This option uses the WIMSE Identity Token (ref TBD) to sign the request and optionally, the response.
+This specification defines a profile of the Message Signatures specification {{!RFC9421}}.
+
+The request is signed as per {{RFC9421}}. The following derived components MUST be signed:
+
+* `@method`
+* `@request-target`
+
+In addition, the following headers MUST be signed when they exist:
+
+* `content-type`
+* `content-digest`
+* `Authorization`
```

Comment:
> should we include `Txn-Token`? A fair number of folks seem to think it might be used to convey transational context info https://www.ietf.org/archive/id/draft-ietf-oauth-transaction-tokens-01.html#name-txn-token-http-header

Yes.